pkg/csi_driver/node.go

```diff
@@ -149,6 +149,10 @@ func (s *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublish
 		fuseMountOptions = joinMountOptions(fuseMountOptions, []string{"hnw-ksa=true", "token-server-identity-provider=" + identityProvider})
 	}
 
+	if enableCloudProfilerForSidecar {
+		fuseMountOptions = joinMountOptions(fuseMountOptions, []string{"enable-clooud-profiler-for-sidecar=" + strconv.FormatBool(enableCloudProfilerForSidecar)})
```

Comment:
create a const for this and also a typo
enable-clooud-profiler-for-sidecar -> enable-cloud-profiler-for-sidecar